climakitae/threshold_tools.py

```diff
@@ -400,77 +405,90 @@ def get_return_value(
         )
 
-        return return_value, conf_int_lower_limit, conf_int_upper_limit
+        return return_variable, conf_int_lower_limit, conf_int_upper_limit
 
-    return_value, conf_int_lower_limit, conf_int_upper_limit = xr.apply_ufunc(
-        return_value,
+    return_variable, conf_int_lower_limit, conf_int_upper_limit = xr.apply_ufunc(
+        _return_variable,
         ams,
         input_core_dims=[["time"]],
         exclude_dims=set(("time",)),
         output_core_dims=[[], [], []],
     )
 
-    return_value = return_value.rename("return_value")
-    new_ds = return_value.to_dataset()
+    return_variable = return_variable.rename(data_variable)
+    new_ds = return_variable.to_dataset()
     new_ds["conf_int_lower_limit"] = conf_int_lower_limit
     new_ds["conf_int_upper_limit"] = conf_int_upper_limit
 
     if multiple_points:
         new_ds = new_ds.unstack("allpoints")
 
-    new_ds["return_value"].attrs["return period"] = "1-in-{}-year event".format(
-        str(return_period)
-    )
+    new_ds.attrs = ams_attributes
+
+    if data_variable == "return_value":
+        new_ds["return_value"].attrs["return period"] = f"1-in-{arg_value}-year event"
+    elif data_variable == "return_prob":
+        threshold_unit = ams_attributes["units"]
+        new_ds["return_prob"].attrs[
+            "threshold"
+        ] = f"exceedance of {arg_value} {threshold_unit} event"
+        new_ds["return_prob"].attrs["units"] = None
+    elif data_variable == "return_period":
+        return_value_unit = ams_attributes["units"]
+        new_ds["return_period"].attrs[
+            "return value"
+        ] = f"{arg_value} {return_value_unit} event"
+        new_ds["return_period"].attrs["units"] = "years"
+
     new_ds["conf_int_lower_limit"].attrs[
         "confidence interval lower bound"
     ] = "{}th percentile".format(str(conf_int_lower_bound))
     new_ds["conf_int_upper_limit"].attrs[
         "confidence interval upper bound"
     ] = "{}th percentile".format(str(conf_int_upper_bound))
 
-    new_ds.attrs = ams_attributes
-    new_ds.attrs["distribution"] = "{}".format(str(distr))
+    new_ds.attrs["distribution"] = f"{distr}"
     return new_ds
 
 
-def get_return_prob(
+def get_return_value(
     ams,
-    threshold,
+    return_period=10,
     distr="gev",
     bootstrap_runs=100,
     conf_int_lower_bound=2.5,
     conf_int_upper_bound=97.5,
     multiple_points=True,
 ):
-    """Creates xarray Dataset with return probabilities and confidence intervals from maximum series
+    """Creates xarray Dataset with return values and confidence intervals from maximum series
 
-    Returns dataset with return probabilities and confidence intervals from maximum series.
+    Returns dataset with return values and confidence intervals from maximum series.
 
     Parameters
     ----------
     ams: xarray.DataArray
-    threshold: float
+    return_period: float
     distr: str
     bootstrap_runs: int
     conf_int_lower_bound: float
```

Comment:
Can you add a brief description of what each input is? Just for user-facing functions I think it's nice to have a bit more info about what each argument is beyond just the object type. For example, something like this: 
```
Parameters
------------
ams: xarray.DataArray
   Annual maximum data, outputted by the function get_ams()
```

Reply:
yes will do!